packages/bigstringaf/bigstringaf.0.5.1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/bigstringaf"
+bug-reports: "https://github.com/inhabitedtype/bigstringaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/bigstringaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {build & >= "1.6"}
```
According to your `dune-package` file, this package requires dune >= 1.6

Reply:
Fixed in inhabitedtype/bigstringaf#38. I don't know if dune does validation, but the build files certainly don't use any new features.

Reply:
ok thanks! This PR will need to be updated though